build.sbt

```diff
@@ -19,9 +20,10 @@ lazy val root = (project in file("."))
       "org.scalameta"   %% "munit"               % MunitVersion           % Test,
       "org.typelevel"   %% "munit-cats-effect-2" % MunitCatsEffectVersion % Test,
       "ch.qos.logback"  %  "logback-classic"     % LogbackVersion,
-      "org.scalameta"   %% "svm-subs"            % "20.2.0"
-    ),
-    addCompilerPlugin("org.typelevel" %% "kind-projector"     % "0.13.0" cross CrossVersion.full),
-    addCompilerPlugin("com.olegpy"    %% "better-monadic-for" % "0.3.1"),
+      "org.scalameta"   %% "svm-subs"            % "20.2.0",
```

Comment:
I did not, I didn't pay that close attention to build.sbt, as long as it worked (not optimal solution obviously, but wanted to finish the task as soon as I could since I have to prepare for my end-of-term exams)